### ISSUE TITLE
If i have genotype data in Numeric format, do I need to adjust the order of genotype file to be consistent with phenotype and map file?

### ISSUE THREAD

**Issue by biolittleboy** [xiaolei-lab/rMVP#62]
Hi, Dr. Liu. I am running Gene-PAV-GWAS. All files need to be generated manually. Does the order of the rows of the ‘Numeric.txt’ and ‘Map.txt’ files need to be consistent? and does the order of the columns of the ‘Numeric.txt’ file need to be consistent with the order of the rows of the ‘Phenotype.txt’ file?
Thanks in advance for the help.

**Comment by hyacz** [comment]
Yes, they need to be consistent. One row of the genotype file represents a marker, and one column represents an individual. The order of the rows needs to be the same as the map file, and the order of the columns needs to be the same as the phenotype file.
You can check https://github.com/xiaolei-lab/rMVP#numeric for detailed instructions
